typings/styled-components.d.ts

```diff
@@ -16,9 +16,10 @@ export type ThemedOuterStyledProps<P, T> = P & {
 };
 export type OuterStyledProps<P> = ThemedOuterStyledProps<P, any>;
 
+export type FalseyValue = undefined | null | false | "";
```

Comment:
Source: [src/utils/flatten.js](https://github.com/styled-components/styled-components/blob/85617db951010592282c4b9cf5287a12e539d777/src/utils/flatten.js#L34)

Reply:
I don't mind either way, I will change it now.